pkg/collector/corechecks/servicediscovery/events.go

```diff
@@ -0,0 +1,132 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package servicediscovery
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/DataDog/datadog-agent/comp/core/hostname"
+	"github.com/DataDog/datadog-agent/comp/core/hostname/hostnameimpl"
+	"github.com/DataDog/datadog-agent/comp/forwarder/eventplatform"
+	"github.com/DataDog/datadog-agent/pkg/aggregator/sender"
+	pkgconfig "github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+type eventType string
+
+const (
+	eventTypeStartService     = "start-service"
+	eventTypeEndService       = "end-service"
+	eventTypeHeartbeatService = "heartbeat-service"
+)
+
+type eventPayload struct {
+	APIVersion          string    `json:"api_version"`
+	NamingSchemaVersion string    `json:"naming_schema_version"`
+	RequestType         eventType `json:"request_type"`
```

Comment:
@rarguelloF I talked to @jonbodner yesterday and he confirmed that he is OK with removing api_version and request_type from the payload and only keeping them in the main event struct.

Reply:
removed those fields 👍